src/extensions/core-api/stores.ts

```diff
@@ -1,8 +1,4 @@
 export { ExtensionStore } from "../extension-store";
-
-export { clusterStore, Cluster, ClusterStore } from "../stores/cluster-store";
-export type { ClusterModel, ClusterId } from "../stores/cluster-store";
-
-export { workspaceStore, Workspace, WorkspaceStore } from "../stores/workspace-store";
-export type { WorkspaceId, WorkspaceModel } from "../stores/workspace-store";
-
+export { KubernetesCluster, KubernetesClusterCategory } from "../../common/catalog-entities/kubernetes-cluster";
+export { catalogCategoryRegistry as catalogCategories } from "../../common/catalog-category-registry";
+export { catalogEntities } from "./catalog";
```

Comment:
Catalogue isn't a store, it is completely on demand.

Reply:
Yes, I just didn't have good idea where to put this. Maybe it should be top-level module in extension API? (like `Catalog.catalogEntities`)

Reply:
I would prefer a new top level.